make.go

```diff
@@ -21,6 +22,24 @@ func commandOutput(name string, args ...string) string {
 	return strings.TrimSpace(string(out))
 }
 
+// List all the go files in the current directory, excluding this file
+func listFilesToBuild() []string {
```

Comment:
It does yeah. Basically since we're using a new file `log.go` in the `main` package, we need to add it to the list of arguments of the `go build` command, hence this function that lists the go files in the project root dir. (before this we only had the `gohai.go` file to pass to the `go build` command, so it was hardcoded)